ui/components/app/terms-of-use-popup/terms-of-use-popup.js

```diff
@@ -0,0 +1,1160 @@
+import React, { useContext, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { I18nContext } from '../../../contexts/i18n';
+import Popover from '../../ui/popover';
+import {
+  AlignItems,
+  FLEX_DIRECTION,
+  TextVariant,
+  Color,
+} from '../../../helpers/constants/design-system';
+import {
+  Text,
+  Button,
+  BUTTON_TYPES,
+  ButtonLink,
+  Label,
+} from '../../component-library';
+import Box from '../../ui/box';
+import CheckBox from '../../ui/check-box/check-box.component';
+
+export default function TermsOfUsePopup({ onAccept }) {
+  const t = useContext(I18nContext);
+  const [isTermsOfUseChecked, setIsTermsOfUseChecked] = useState(false);
+  const [shouldShowScrollButton, setShouldShowScrollButton] = useState(true);
+
+  const popoverRef = useRef();
+  const bottomRef = React.createRef();
+
+  const handleScrollDownClick = (e) => {
+    e.stopPropagation();
+    bottomRef.current.scrollIntoView({
+      behavior: 'smooth',
+    });
+    setShouldShowScrollButton(false);
+  };
+
+  return (
+    <Popover
+      className="terms-of-use__popover"
+      popoverRef={popoverRef}
+      title={t('termsOfUseTitle')}
+      showScrollDown={shouldShowScrollButton}
+      onScrollDownButtonClick={handleScrollDownClick}
+      footerProps={{
+        justifyContent: AlignItems.center,
+        flexDirection: FLEX_DIRECTION.COLUMN,
+      }}
+      footer={
+        <>
+          <Button
+            type={BUTTON_TYPES.PRIMARY}
+            className="terms-of-use__button"
+            onClick={onAccept}
+            disabled={!isTermsOfUseChecked}
+            data-testid="terms-of-use-accept-button"
+          >
+            {t('accept')}
+          </Button>
+          <Text as="p" marginTop={4} className="terms-of-use__footer-text">
+            {t('termsOfUseFooterText')}
+          </Text>
```

Comment:
The `Text` color seems to be different from the Figma design. Please update this
![Screenshot 2023-04-11 at 7 49 08 PM](https://user-images.githubusercontent.com/39872794/231192349-3ddc743e-4b3f-423f-94c9-3157e9690761.jpg)